### PR TITLE
Added transpose of a Matrix C++

### LINKED ISSUE #9: Transpose of a matrix
I'm willing to contribute and write transpose of a matrix. For hactober fest

### PR DESCRIPTION
**Pull request by rafu01**
I've added transpose of a matrix in C++ with proper comments in the code
Check and review 
thank you

### CODE REVIEW

**Comment by TomaszWychocki** [comment]
Heeey wait a minute. Add your implementation into existing class. Do not add new file.

**Comment by rafu01** [comment]
Okay I'm working on it

**Comment by rafu01** [comment]
Can you check now? I've added in the previous class file and also because of vscode auto-indentation some spaces got deleted that's why it shows conflict.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
```
Comment (TomaszWychocki) [inline .vscode/settings.json:1]: Remove this file

File: MatrixAlgorithms/Matrix2d.cpp [hunk 2]
```diff
@@ -190,24 +186,41 @@ std::vector<double> Matrix2d::toVector() const
...
+        std::vector<double> row_t;
+        for (int j = 0; j < rows; j++)
+        {
+            double cell = getValue(j, i);
+            row_t.push_back(cell);
+        }
+        mat_t.push_back(row_t);
+    }
+    return mat_t;
+}
```
Comment (TomaszWychocki) [inline MatrixAlgorithms/Matrix2d.cpp]: This method should modify current object

File: MatrixAlgorithms/MatrixAlgorithms.cpp [hunk 3]
```diff
@@ -213,6 +212,11 @@ void TEST_010_subtraction_2()
     }
     assert(isException);
 }
+std::vector<std::vector<double>> TEST_011_transpose()
+{
+    Matrix2d mat1 = {{2, 3, 4, 8}, {5, 6, 7, 9}};
+    return mat1.transpose();
+}
```
Comment (TomaszWychocki) [inline MatrixAlgorithms/MatrixAlgorithms.cpp]: Create a test case. This function doesn't test anything. Use 'assert' function to test you code.

**Review by TomaszWychocki — COMMENTED** [review]
There are severtal tasks to be done:
- Pull latest changes to resolve all conflicts.
- Remove all unneccessary changes generated by VS code
- Add new test case to verify your code.

**Pull request closed without merging**